build.gradle

```diff
@@ -130,6 +130,14 @@ repositories {
   maven {
     url 'https://dl.bintray.com/palantir/releases' // docker-compose-rule is published on bintray
   }
+
+  maven {
+    url "https://dl.bintray.com/hmcts/hmcts-maven"
+  }
+
+  maven {
+    url 'https://repo.spring.io/libs-milestone'
+  }
```

Comment:
I assume this is needed for `springHateoas`. But version is `.RELEASE` suffixed so maybe not a milestone and available in jcentre?